zerver/lib/validator.py

```diff
@@ -72,10 +82,9 @@ def f(var_name, val):
     return f
 
 def check_dict(required_keys):
-    # required_keys is a list of tuples of
-    # key_name/validator
-
```

Comment:
Good idea. The module docstring has notes on how to use `check_list` and `check_dict`, but I think those might be better as docstrings of the functions themselves. Any objections to relocating those notes?